lleida_net/click_sign/client.py

```diff
@@ -143,6 +143,49 @@ def status(self, signatory_id):
 
         raise NotFoundSignatureException()
 
+    def get_document_stamp(self, data):
+        """
+        It tries to reach the signatory stamp of an already signed signature.
+        If not, raises an exception
+        """
+        assert isinstance(data, dict), "Data must be a dict"
+        data['file_group'] = "SIGNATORY_STAMP"
+
+        try:
+            signatory_data = schema.SignatoryStampSchema().dump(data).data
+            response = self.api.post(
+                resource="get_document", json=signatory_data)
+            validated_response = schema.GetDocumentSchema().load(response)
+
+        except Exception as e:
+            raise NotValidSignatureSchemaException(str(e))
```

Comment:
Maybe it's a good idea to define an specific Exception to be able to identify this specific error.

See https://github.com/gisce/lleida_net_api/blob/get_signed_document/lleida_net/click_sign/client.py#L27